app/uk/gov/hmrc/thirdpartydeveloperfrontend/domain/models/applications/Capabilities.scala

```diff
@@ -49,7 +49,9 @@ object Capabilities {
     def hasCapability(app: ApplicationWithCollaborators) = true
   }
 
-  case object SupportsSubscriptions extends StandardAppCapability
+  case object SupportsSubscriptions extends Capability {
```

Comment:
Are we sure we didn't want to go down the route of SupportsReadingSubscriptions and EditSubscriptions as two separate capabilities?

Reply:
Maybe, seems like its doubling up where the `Locking` behavior is (which is driven from the app domain)